Cargo.toml

```diff
@@ -22,7 +22,7 @@ default = [
 
 [dependencies]
 winit = {version = "0.25.0", features = ["web-sys"]}
-bevy = { version="0.5.0", default-features=false }
+bevy = { git = "https://github.com/bevyengine/bevy", default-features=false }
```

Comment:
i think you will need to specify the branch name(at least, i have to when trying to use the git repo directly). cargo assumes `master` will be the default branch and bevy's is `main`

```suggestion
bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features=false }
```

Reply:
Thank you, I wasn't aware of that.